$NetBSD: patch-doc_socket++.texi,v 1.1 2022/04/25 16:17:48 tnn Exp $

socket++.texi:2337: raising the section level of @subsection which is too low
socket++.texi:2452: raising the section level of @subsection which is too low

--- doc/socket++.texi.orig	2002-11-25 12:31:56.000000000 +0000
+++ doc/socket++.texi
@@ -2334,7 +2334,6 @@ an @code{echo} object is a smart pointer
 
 @end table
 
-@subsection tsecho.C
 @example
 // echo server. Serves clients at port 4000.
 #include <echo.h>
@@ -2449,7 +2448,6 @@ recipient previously established through
 
 @end table
 
-@subsection tcsmtp.C
 @example
 // smtp client.
 // The president sends a message to gs4t@@virginia.edu.
